mono_repo/lib/src/github_config.dart

```diff
@@ -27,6 +33,21 @@ class GitHubConfig {
       };
 }
 
+@JsonSerializable(createToJson: false, disallowUnrecognizedKeys: true)
+class GitHubWorkflow {
+  @JsonKey(nullable: false)
+  final String name;
+  @JsonKey(nullable: false)
+  final Set<String> stages;
+
+  GitHubWorkflow(this.name, this.stages) {
+    // TODO: stages shouldn't be empty
```

Comment:
We should file issues for any todos just to help keep track of them, even if its pretty low priority like this one

Reply:
yup

Reply:
fixed it